parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -52,6 +52,9 @@ ERR_CIS_ASSIGN_ROOT_PW=111 # Error assigning root password in CIS enforcement
 ERR_CIS_ASSIGN_FILE_PERMISSION=112 # Error assigning permission to a file in CIS enforcement
 ERR_CIS_COPY_FILE=113 # Error writing a file to disk for CIS enforcement
 ERR_CIS_APPLY_PASSWORD_CONFIG=115 # Error applying CIS-recommended passwd configuration
+ERR_AZURE_STACK_GET_ARM_TOKEN=116 # Error generating a token to use with Azure Resource Manager
```

Comment:
let's allocate specific range for Azure Stack specific errors, eg. 120-130. Can you make these error codes 120, 121 and 122?